px-map-behavior-marker.es6.js

```diff
@@ -345,7 +342,12 @@
       return {
         type: this.type,
         symbol: this.symbolClass,
-        styleScope: this.isShadyScoped() ? this.getShadyScope() : undefined
+        icon: this.icon,
+        styleScope: this.isShadyScoped() ? this.getShadyScope() : undefined,
```

Comment:
I misread that as 3 `getComputedStyle` calls as opposed to `getComputedStyleValue`. We could still call `getComputedStyle` only once, the returned object is live and will automatically be updated if the style changes, however we still have to call `getPropertyValue` on it and the `getComputedStyleValue` seem to cater for more scenarios as seen in Polymer code. Long story short you can ignore this :)